d2l-icon-button.html

```diff
@@ -1,23 +1,20 @@
 <link rel="import" href="../polymer/polymer.html">
-<link rel="import" href="../d2l-offscreen/d2l-offscreen-shared-styles.html">
 <link rel="import" href="d2l-icon.html">
 <link rel="import" href="d2l-icon-button-shared-styles.html">
 <link rel="import" href="d2l-icon-button-behavior.html">
 
 <dom-module id="d2l-icon-button">
 	<template strip-whitespace>
```

Comment:
Now that the button is going inside, you may want to apply `display: inline-block` to the new `host` element.